src/utils/worklets/chat/messenger/composer.cljs

```diff
@@ -6,14 +6,6 @@
   [chat-list-scroll-y composer-focused? window-height]
   (.scrollDownButtonOpacity ^js worklets chat-list-scroll-y composer-focused? window-height))
 
-(defn jump-to-button-opacity
-  [scroll-down-button-opacity-sv composer-focused?]
-  (.jumpToButtonOpacity ^js worklets scroll-down-button-opacity-sv composer-focused?))
```

Comment:
Can we remove the JS functions `jumpToButtonOpacity` and `jumpToButtonPosition` as well? I couldn't find usages.